frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/TicfCriHandler.java

```diff
@@ -0,0 +1,98 @@
+package uk.gov.di.authentication.frontendapi.lambda;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import org.apache.logging.log4j.LogManager;
+import uk.gov.di.authentication.entity.TICFCRIRequest;
+import uk.gov.di.authentication.shared.services.CloudwatchMetricsService;
+import uk.gov.di.authentication.shared.services.ConfigurationService;
+import uk.gov.di.authentication.shared.services.SerializationService;
+
+import java.io.IOException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.net.http.HttpTimeoutException;
+import java.time.Duration;
+import java.util.Map;
+
+import static java.lang.String.format;
+
+public class TicfCriHandler implements RequestHandler<TICFCRIRequest, Void> {
+
+    private final HttpClient httpClient;
+    private final ConfigurationService configurationService;
+    private final CloudwatchMetricsService cloudwatchMetricsService;
+
+    private static final SerializationService serialisationService =
+            SerializationService.getInstance();
+
+    public TicfCriHandler(
+            HttpClient httpClient,
+            ConfigurationService configurationService,
+            CloudwatchMetricsService cloudwatchMetricsService) {
+        this.httpClient = httpClient;
+        this.configurationService = configurationService;
+        this.cloudwatchMetricsService = cloudwatchMetricsService;
+    }
+
+    public TicfCriHandler(ConfigurationService configurationService) {
+        this.httpClient = HttpClient.newHttpClient();
+        this.configurationService = configurationService;
+        this.cloudwatchMetricsService = new CloudwatchMetricsService();
+    }
+
+    public TicfCriHandler() {
+        this(ConfigurationService.getInstance());
+    }
+
+    private static final org.apache.logging.log4j.Logger LOG =
+            LogManager.getLogger(TicfCriHandler.class);
+
+    @Override
+    public Void handleRequest(TICFCRIRequest input, Context context) {
+        LOG.info("received request to TICF CRI Handler");
+        var environmentForMetrics = Map.entry("Environment", configurationService.getEnvironment());
+        try {
+            var response = sendRequest(input);
+            var statusCode = String.valueOf(response.statusCode());
+            LOG.info("Response received from TICF CRI Service with status %s", statusCode);
```

Comment:
Similar consideration about log level here as well.

Reply:
So for now I want to keep this in as info since we don't have debug configured (I don't think) in lower envs and for now this is one of the main things that tells us what's going on or not, but later on we can switch to debug